python/py-coolprop/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        CoolProp CoolProp 6f4dfa82c1232f4827a81040284262a7099ed3fe
```

Comment:
this should take a stable released version, not a git hash

Reply:
also, please take the sources from PyPI instead, which avoids fetching submodules that we like to avoid whenever possible. I haven't looked in detail what it's trying to fetch/build from submodules, but at least some of these packages are part of MacPorts already and these should preferably be used instead.


In other words: remove the `github` PG and do instead:
```
name                py-coolprop
python.rootname     CoolProp
version             6.4.1
```

of course that will require updating the checksums and let's see how that goes.

Reply:
I just tried fetching 6.4.1 from PyPI as you suggest, but I get the same result I had when I originally tried to install from pip. There are several compile errors. It is my understanding that those have been fixed in the Github repository but haven't been pushed to PyPI. This is the reason why I attempted to compile from source using MacPorts in the first place. I chose this git hash because it was the most recent when I initially compiled manually from source, and I fully tested it.

It seems the latest stable tag is v6.4.1 which is presumably the same as the one from PyPI.

Reply:
then it seems time for upstream to release a new version with these fixes ;) 

We still don't want a semi-random git hash; ideally one would pick version 6.4.1 from PyPI and add the fixes as a patch to get it to build if the changes are small.

Reply:
I'll try and take a look one of these days...

Reply:
> I'll try and take a look one of these days...

Renee, where do things stand with this?

Reply:
FYI, I have reached out to upstream and asked them to release a new version.
See: https://github.com/CoolProp/CoolProp/issues/2178